docker/ci/dockerfiles/current/build.centos7.opensearch.x64.arm64.dockerfile

```diff
@@ -97,6 +97,10 @@ RUN pip3 install pip==21.3.1
 RUN pip3 install cmake==3.21.3
 RUN pip3 install awscli==1.22.12
 
+# Install osslsigncode tool
+RUN yum clean all && wget https://download-ib01.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/o/osslsigncode-2.2-1.el8.x86_64.rpm && \
+    yum install -y osslsigncode-2.2-1.el8.x86_64.rpm 
+
```

Comment:
We should avoid using wget if not need to, use `curl -SLO` instead.
Also why are we adding this tool specifically to the OS build image?

Thanks.

Reply:
Also this would not work as you are only installing the X64 version of the rpm, the ARM64 will fail after all.

Reply:
I can remove it from build image if it is not required now. I dont know what image we are using for signing today. https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/promotion/promote-artifacts.jenkinsfile#L45

Any idea?

Reply:
Made the necessary changes. Please review. Thanks!